LabExT/View/LiveViewer/Cards/PowerMeterCard.py

```diff
@@ -28,14 +27,16 @@ class PowerMeterCard(CardFrame):
         'powermeter range': MeasParamInt(value=0, unit='dBm'),
         # integration time of power meter
         'powermeter averagetime': MeasParamFloat(value=0.1, unit='s'),
-        # integration time of power meter
+        # center wavelength of observed signal
         'powermeter wavelength': MeasParamFloat(value=1550.0, unit='nm'),
+        # which channels to interrogate
+        'channels to plot (comma separated)': MeasParamString(value='1')
```

Comment:
I don't think we could use `MeasParamList` here. I want the user to be able to select multiple channel designators too, which is not possible with a list / dropdown (which selects 1-out-of-N instead of M-out-of-N).

Reply:
Ah you're right, I misunderstood `MeasParamList`, sorry.